storage/lmdb/Cargo.toml

```diff
@@ -8,7 +8,9 @@ license = "MIT"
 storage-core = { path = '../core' }
 utils = { path = '../../utils' }
 
-lmdb-rkv = "0.14"
+[dependencies.lmdb-mintlayer]
```

Comment:
Why not use braces like others?

Reply:
I just found the line too long but can change it back, no strong opinions here

Reply:
Personally I prefer the "single line" dependency format, but in the p2p we have a lot of dependencies in the "multiline" format.

Reply:
Would be great if you could change it... it makes global searches much easier when everything is homogeneous. I want to change them also in p2p.

Reply:
ok